c++/src/kj/async-io.c++

```diff
@@ -1261,8 +1264,8 @@ private:
 #if __cplusplus < 201402L
       inline constexpr Need(uint64_t minBytes, uint64_t maxBytes)
           : minBytes(minBytes), maxBytes(maxBytes) {}
-      // TODO(cleanup): This constructor is only here to support brace initialization in C++11. It
-      //   should be removed once we upgrade to C++14.
+      // TODO(cleanup): This constructor is only here to support brace initialization in gcc 4.9. It
```

Comment:
:man_facepalming: If you feel like bumping the CI to a newer GCC now I'm cool with that. Or if you'd rather leave it, that's fine too.